JXHTTP/JXHTTPOperation.h

```diff
@@ -303,4 +303,16 @@ typedef NSURLRequest * (^JXHTTPRedirectBlock)(JXHTTPOperation *operation, NSURLR
  */
 + (void)setNetworkActivityIndicatorManager:(id <JXNetworkActivityIndicatorManager>)networkActivityIndicatorManager;
 
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_2_0
+
+/**
+ *  The number of active operations.
+ *  This is used to determine whether or not the activity indicator of the device is shown.
```

Comment:
Ah interesting, and change the type to `BOOL` as well? This probably makes more sense as I think that is all _I need_ to know.